dbms/src/Debug/astToExecutor.cpp

```diff
@@ -1525,4 +1525,18 @@ ExecutorPtr compileJoin(size_t & executor_index, ExecutorPtr left, ExecutorPtr r
     return join;
 }
 
+
+ExecutorPtr compileExchangeSender(ExecutorPtr input, size_t & executor_index)
+{
+    ExecutorPtr exchange_sender = std::make_shared<mock::ExchangeSender>(executor_index, input->output_schema, tipb::PassThrough);
```

Comment:
In many cases it is not tipb::PassThrough, but tipb::HashXXX.
need to be specified by the user

Reply:
I will fix it